ledgerautosync/cli.py

```diff
@@ -55,7 +55,7 @@ def find_ledger_file(ledgerrcpath=None):
         return None
 
 
-def print_results(converter, ofx, ledger, txns, args):
+def print_results(converter, ofx, ledger, txns, args, account_idx=0):
```

Comment:
I think this function would be better if we passed in `account` instead of `ofx` and `account_idx`.

```diff
diff --git a/ledgerautosync/cli.py b/ledgerautosync/cli.py
index a007c59..806e798 100755
--- a/ledgerautosync/cli.py
+++ b/ledgerautosync/cli.py
@@ -55,7 +55,7 @@ found."""
         return None
 
 
-def print_results(converter, ofx, ledger, txns, args, account_idx=0):
+def print_results(converter, account, ledger, txns, args):
     """
     This function is the final common pathway of program:
 
@@ -69,18 +69,18 @@ def print_results(converter, ofx, ledger, txns, args, account_idx=0):
         if (not(ledger.check_transaction_by_id
                 ("ofxid", converter.mk_ofxid(AUTOSYNC_INITIAL))) and
                 not(ledger.check_transaction_by_id("ofxid", ALL_AUTOSYNC_INITIAL))):
-            print(converter.format_initial_balance(ofx.accounts[account_idx].statement))
+            print(converter.format_initial_balance(account.statement))
     for txn in txns:
         print(converter.convert(txn).format(args.indent))
     if args.assertions:
-        print(converter.format_balance(ofx.accounts[account_idx].statement))
+        print(converter.format_balance(account.statement))
 
     # if OFX has positions use these to obtain commodity prices
     # and print "P" records to provide dated/timed valuations
     # Note that this outputs only the commodity price,
     # not your position (e.g. # shares), even though this is in the OFX record
-    if hasattr(ofx.accounts[account_idx].statement, 'positions'):
-        for pos in ofx.accounts[account_idx].statement.positions:
+    if hasattr(account.statement, 'positions'):
+        for pos in account.statement.positions:
             print(converter.format_position(pos))
 
 def make_ofx_converter(account,
@@ -138,7 +138,7 @@ def sync(ledger, accounts, args):
                                                hardcodeaccount=None,
                                                shortenaccount=args.shortenaccount,
                                                security_list=SecurityList(ofx))
-                print_results(converter, ofx, ledger, txns, args)
+                print_results(converter, acct, ledger, txns, args)
         except KeyboardInterrupt:
             raise
         except BaseException:
@@ -156,8 +156,7 @@ def import_ofx(ledger, args):
         sys.stderr.write("number of account name (--account) does not match the number of accounts in the OFX file\n")
         exit(1)
 
-    account_idx = 0
-    for account in ofx.accounts:
+    for account_idx, account in enumerate(ofx.accounts):
         txns = sync.filter(
             account.statement.transactions,
             account.account_id)
@@ -191,8 +190,7 @@ def import_ofx(ledger, args):
                                  hardcodeaccount=args.hardcodeaccount, #TODO
                                  shortenaccount=args.shortenaccount, #TODO
                                  security_list=security_list)
-        print_results(converter, ofx, ledger, txns, args, account_idx)
-        account_idx += 1
+        print_results(converter, account, ledger, txns, args)
 
 
 def import_csv(ledger, args):
```